spec/graphql/mutations/incomes/create_income_request_spec.rb

```diff
@@ -30,7 +30,7 @@
       expect(data).to have_key(:date)
 
       expect(data[:userId]).to eq(user.id.to_s)
-      expect(data[:source]).to eq("Paycheck")
+      expect(data[:source]).to eq("paycheck")
```

Comment:
This all looks well tested in the graphql mutation specs. I can see that it's all working, but I'm wondering if we should add model testing for some of the methods